napari/components/experimental/monitor/_service.py

```diff
@@ -0,0 +1,175 @@
+"""MonitorService class.
```

Comment:
We should add more about this being Python 3.9 only as I think most people might miss that on first look. Not sure if here is the best place or elsewhere. It could come later if we merge this first, but something to keep in mind

Reply:
Okay I put a line about 3.9 right below that.

Reply:
Maybe needs to be other places too though.